linked-environments.html

```diff
@@ -0,0 +1,201 @@
+                </tr>
+                <tr>
+              {% endif %}
+                <td class="table-cell">
+                <table class="table">
+                <tr>
+                <td class="table-cell"><a href="{{ item.url }}" target="_blank"><img src="{{ item.image }}" alt="{{ item.name }}"/></a></td>
+                </tr>
+                <tr>
+                <td class="table-cell">{{ item.description }}</td>
+                </tr>
+                <tr>
+                <td class="table-cell">
+                {% for l in links %}
+                  <a href="{{ l.url }}" target="_blank"><img src="{{ l.image }}" alt="{{ l.name }}"/></a>
+                {% endfor %}
+                </td>
+                </tr>
+                </table>
+                </td>
+              {% assign count = count | plus: 1 %}
+              {% endif %}
+           {% endfor %}
+           </tr>
+           <table>
+        </div>
+
+        <div class="row column text-center">
+            <h3>Using tools to explore IDR</h2>
+        </div>
+        <div class="row align-right">
+            <table class="table">
+            {% assign count = 0 %}
+            <tr>
+            {% for item in items %}
+              {% if item.name contains "Tools" %}
+              {% assign links=item.breadcrumb %}
+              {% if count == 3 %}
+                {% assign count = 0 %}
+                </tr>
+                <tr>
+              {% endif %}
+                <td class="table-cell">
+                <table class="table">
+                <tr>
+                <td class="table-cell"><a href="{{ item.url }}" target="_blank"><img src="{{ item.image }}" alt="{{ item.name }}"/></a></td>
+                </tr>
+                <tr>
+                <td class="table-cell">{{ item.description }}</td>
+                </tr>
+                <tr>
+                <td class="table-cell">
+                {% for l in links %}
+                  <a href="{{ l.url }}" target="_blank"><img src="{{ l.image }}" alt="{{ l.name }}"/></a>
+                {% endfor %}
+                </td>
+                </tr>
+                </table>
+                </td>
+              {% assign count = count | plus: 1 %}
+              {% endif %}
+           {% endfor %}
+           </tr>
+           <table>
+        </div>
+        <!-- end Notebooks section -->
+
+
+
+       <hr class="whitespace">
+        <div class="row column text-center">
+            <h2>Developing your own</h2>
+        </div>
+        <div class="row">
+            <div class="small-10 small-centered medium-10 medium-centered columns">
+                <div class="row horizontal">
+                    <div>
+                      <h1 class="text-center">GitHub login</h1>
+                      <p>
+                        If you are interested in developing your own data analysis notebooks on our analysis platform please contact us with your GitHub username.
```

Comment:
This is only available for the BioImageArchive BinderHub platform right?

Reply:
yes